android/src/main/java/net/no_mad/tts/TextToSpeechModule.java

```diff
@@ -158,6 +156,7 @@ public void onBeginSynthesis(String utteranceId, int sampleRateInHz, int audioFo
                             tts.stop();
                         } else {
                             try {
+                                
```

Comment:
I think the call to `forceSpeakerRoute` may have been accidentally been removed with the latest changes?

Reply:
idk what is going on in my IDE.... let me fix that